### DISCUSSION TITLE
Start OnNewIntent when app is closed

### DISCUSSION THREAD

**Discussion by TechnikKTK** [thudugala/Plugin.LocalNotification#244] [General]
Hi. I very nice to using this package. but i not understand why its working when app is closed. I tap on notify, but app opening is standart view and not proccess my code (popUp message)

**Comment by TechnikKTK** [comment]
![image](https://user-images.githubusercontent.com/71823154/149753089-64e4d5be-0f59-41a9-82a8-22ff759bc217.png)

**Comment by TechnikKTK** [comment]
![image](https://user-images.githubusercontent.com/71823154/149753178-488790f0-da9d-47fa-b99e-c5673800f735.png)

**Comment by thudugala** [comment]
@TechnikKTK  try version 10.1.1
